armi/reactor/reactors.py

```diff
@@ -1724,13 +1713,6 @@ def createAssemblyOfType(self, assemType=None, enrichList=None, cs=None):
                     # therefore breaks the burnup metric.
                     b.adjustUEnrich(enrich)
 
-        # see if there are tracked assemblies and add this to the list if so
-        if self._trackAssems:
-            self.cfp.add(a)
```

Comment:
This is where the usage of the `cfp` was causing issues. Outside of this, the `cfp` was not being utilized in the framework or TerraPower plugin code.

Reply:
I don't think this can cause duplication since it will just hold pointers to the assemblies. But in any case, we do want to get rid of the (confusing) CFP so we should accept this as progress in that direction.

Reply:
It's not really duplication that we are worried about, but rather lack of it. If `a` goes into `self.cfp`, then the same `a` gets returned to the caller (almost certainly to be `add()`ed to something else), unless the caller is explicitly `deepcopy`ing the returned assembly, we end up with two references to the same Assembly.